i3ipc.py

```diff
@@ -413,6 +413,15 @@ def __init__(self, data, parent, conn):
             else:
                 setattr(self, attr, None)
 
+        # Make sure marks is never None (use []) to simplify client code. Older
+        # versions of i3 used mark instead of marks, so collapse that into
+        # marks to provide a consistent API.
+        if not self.marks:
```

Comment:
No, I'd rather make this consistent with the new api across any i3 version since it's a very easy translation.

The inconsistency happens like this:

new api:

```python
con.mark # None
con.marks # [ list of marks... ]
```

old api:

```python
con.mark # attribute error
con.marks # singleton list
```

`con.mark` should not be a property of the Con and it should always throw an attribute error.